dotnet/samples/KernelSyntaxExamples/Example15_TextMemoryPlugin.cs

```diff
@@ -204,7 +203,7 @@ private static async Task RunWithStoreAsync(IMemoryStore memoryStore, Cancellati
 
         // Save a memory with the Kernel
         Console.WriteLine("Saving memory with key 'info5': \"My family is from New York\"");
-        await kernel.InvokeAsync(memoryFunctions["Save"], new ContextVariables()
+        await kernel.InvokeAsync(memoryFunctions["Save"], new KernelFunctionArguments()
```

Comment:
Suggest changing the name of the variable to Plugin.

```suggestion
        await kernel.InvokeAsync(memoryPlugin["Save"], new KernelFunctionArguments()
```